Develop1

ZaurSeyfulaev opened the pull request.

ZaurSeyfulaev committed:
Добавлены тесты по дипломной работе 1

ZaurSeyfulaev committed:
Добавлен jacoco отчет

ZaurSeyfulaev committed:
Поправил тесты по замечаниям и вопросам на вебинаре

ZaurSeyfulaev committed:
Исправил тесты на основе обсуждений.
-убрал зависимость теста от Database;
-написал тест на IngredientType;
-почистил код;
-поправил отчет

pomegranate282922 commented on src/test/java/BurgerTest.java:
```diff
@@ -0,0 +1,88 @@
+        ingredient.getType();
+        ingredient.getName();
+        addIngredientInList(1);
+        assertEquals(true, burger.getReceipt().contains("hot space"));
```
Не полностью протестировано

pomegranate282922 commented on src/test/java/BurgerTest.java:
```diff
@@ -0,0 +1,88 @@
+    private Ingredient ingredient;
+
+
+    @Spy
```
Зачем спай?

pomegranate282922 commented on src/test/java/BurgerTest.java:
```diff
@@ -0,0 +1,88 @@
+
+    }
+
+    public void addIngredientInList(int index) {
```
Зачем public?

pomegranate282922 reviewed (commented):
Хорошо! Немного поправить по замечаниям.

ZaurSeyfulaev committed:
Исправил тесты на основе обсуждений.
-с поомощью куратора понял что надо проверять всю структуру рецепта - исправил
-модификатор изменил на private
-Spy был лишним - убрал.

pomegranate282922 reviewed (approved):
Зачёт!
